src/cinder/Signals.cpp

```diff
@@ -111,6 +111,22 @@ ScopedConnection& ScopedConnection::operator=( ScopedConnection &&rhs )
 	return *this;
 }
 
+ConnectionList::~ConnectionList()
+{
+	clear();
+}
+
+void ConnectionList::add( Connection &&target )
+{
+	mConnections.emplace_back( target );
+}
+
+void ConnectionList::clear()
+{
+	for( auto &conn : mConnections )
```

Comment:
@MikeGitb thanks for the continual suggestions and helping to make this implementation better. 

About `std::vector`, I think we're in the clear for now as I left the vector type in `ConnectionList` as `Connection`, thinking we'd move to it using `ScopedConnection` once our bottom end compilers catch up. Or are you foreseeing some other problem? If the issue is concerning users keeping their own vector of `ScopedConnection`s, I think we can address that in a separate pull request if you see a good way to do it. It won't work across all platforms though, so I'm happy with waiting a bit on that considering we'll be switching to vc140 first thing in the next release cycle.

I pushed the recently discussed changes in c4c736f, so barring the current question this should be good.